app/routes.py

```diff
@@ -1,2 +1,53 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.planet import Planet
 
+planet_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+def validate_planet(id):
+    try:
+        id = int(id)
+    except:
+        abort(make_response({"message": f"planet '{id}' is invalid"}, 400))
+
+    planet = Planet.query.get(id)
+
+    if not planet:
+        abort(make_response({"message": f"Planet {id} not found"}, 404))
+
+    return planet
+
+@planet_bp.route("", methods=["GET", "POST"])
```

Comment:
Be sure to refer to the [second part of creating the get all endpoint](https://learn-2.galvanize.com/cohorts/3430/blocks/1310/content_files/api-3-database-models-read/read-all-books.md#refactor-using-seperate-route-functions) where we avoid using multiple verbs in a single endpoint.

The multiple verb approach is possible, but generally we want to avoid that (single responsibility principle).